test/Elastic.Apm.Extensions.Hosting.Tests/Elastic - Backup.Apm.Extensions.Hosting.Tests.csproj

```diff
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Looking at the name of this file, I think it can be removed?

Reply:
Right, removed it.